openslides_backend/action/base.py

```diff
@@ -109,17 +109,12 @@ def prepare_dataset(self, payload: ActionPayload) -> DataSet:
         """
         raise NotImplementedError
 
-    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+    def get_updated_instances(self, payload: ActionPayload) -> ActionPayload:
```

Comment:
Ah, I didn't see this until now, @reiterl did a similar change in https://github.com/OpenSlides/openslides-backend/pull/251, but maybe the additional base class was overkill. You can fix the conflicts however you want